docs/ed-fi-badges/3-registry-of-ed-fi-badged-products-and-services.md

```diff
@@ -0,0 +1,69 @@
+---
+author: Eric Jansson
+tags: [education Agency,v]
```

Comment:
We'll probably want to rethink the tagging scheme compared to what Confluence had. We'll also need to fix things up, so that we don't have a tag called `v` and `education agency` probably needs to be in quotes due to the space (I would just use `education-agency` instead. Just not sure what value this tag brings.